plugins/node/instrumentation-cucumber/.tav.yml

```diff
@@ -1,3 +1,3 @@
 '@cucumber/cucumber':
-  versions: '^8.0.0 || ^9.0.0'
+  versions: '^8.0.0 || ^9.0.0 || ^10.0.0'
```

Comment:
Although the part of the TAV (test-all-versions) tests that tested cucumber v10 *did* pass, technically v10 dropped support for Node.js v14 and v16 -- https://github.com/cucumber/cucumber-js/releases/tag/v10.0.0 -- so I wonder if we may want a .tav.yml file like this:

```yaml
'@cucumber/cucumber':
  - versions: '^8.0.0 || ^9.0.0'
    node: ">=14 <20"
    commands: npm test
  - versions: '^10.0.0'
    node: ">=18"
    commands: npm test
```